.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -1,5 +1,157 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  IMAGE_WIN_2016: "family/core-windows-2016"
+  IMAGE_WIN_2019: "family/core-windows-2019"
+  IMAGE_WIN_2022: "family/core-windows-2022"
+  DEBIAN_FRONTEND: "noninteractive"
+
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - group: "Filebeat Mandatory Testing"
+    key: "mandatory-tests"
+    if: build.env("GITHUB_PR_TRIGGER_COMMENT") == "filebeat" || build.env("BUILDKITE_PULL_REQUEST") != "false"
+
+    steps:
+      - label: ":ubuntu: Unit Tests"
+        command:
+          - "sudo apt-get update"
+          - "sudo apt-get install -y libsystemd-dev"
+          - "sudo apt install -y python3-pip"
+          - "sudo apt-get install -y python3-venv"
```

Comment:
To avoid duplication, I'd say either:
1) Creating a specific VM with these dependencies in the ci-images repository or
2) Use a script with those dependencies so it can be called in the different steps.

What do you think?

Reply:
After we migrate the beats pipelines, we are planning to create custom images beats-specific. Currently we decided to left it as it is.